src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/AsyncEventHandler_OnElement_ActionEventArgs_Lambda/TestComponent.codegen.cs

```diff
@@ -26,7 +26,13 @@ public class TestComponent : Microsoft.AspNetCore.Components.ComponentBase
         #pragma warning disable 1998
         protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTree.RenderTreeBuilder builder)
         {
-            __o = Microsoft.AspNetCore.Components.EventCallback.Factory.Create<Microsoft.AspNetCore.Components.UIMouseEventArgs>(this, "async (e) => await Task.Delay(10)");
+#nullable restore
+#line 2 "x:\dir\subdir\Test\TestComponent.cshtml"
```

Comment:
Line 2 of `TestComponent.cshtml` is 
`<input onclick=""async (e) => await Task.Delay(10)"" />`.
I assume I am correct in generating a line pragma here?

Reply:
This isn't super correct. See how the lambda isn't padded correctly?

Reply:
Also, I want to know how this bug got put into preview 6. This is a serious-pants regression.

Reply:
https://github.com/aspnet/AspNetCore/issues/8826#issuecomment-478102932